docs/services/data/dataset.md

```diff
@@ -1,153 +0,0 @@
----
```

Comment:
I think we do want to keep a dataset page to have reference information. I am thinking something like this:

```
title: "Dataset"
---

On the [**DATA** tab](https://app.viam.com/data/view) in the Viam app, you can create and manage datasets.
A dataset is a grouping of image data that you can add labels to.
You can use datasets to traing Machine Learning models on. 

{{< alert title="Info" color="info" >}}
Filtered datasets are views and not materialized.
That means the data you are viewing may change as you label and train on the dataset.
If the underlying data matching the filter changes because data is deleted or more data is added, the dataset will also change.
{{< /alert >}}

## Labels

You can label your data with:

- [Bounding boxes](#bounding-boxes), which you can use to [train object detection models](/services/ml/train-model/#train-a-model).
- [Image tags](#image-tags), which you can use to train [image classification models](/services/ml/train-model/#train-a-model).

## API

To interact with datasets programmatically, you use the Dataset API, which supports the following methods:

{{< readfile "/static/include/app/apis/generated/dataset-table.md" >}}

The data client API supports a separate set of methods that allow you to upload and export data to and from the Viam app.
For information about that API, see [Data Client API](/appendix/apis/data-client/).

## Next steps

Link to how-tos.
...



```